### DISCUSSION TITLE
I want to popup a window with the right mouse button, but there is a problem that I can't solve

### DISCUSSION THREAD

**Discussion by sdegongzuo** [slint-ui/slint#4255] [Q&A]
slint version: 1.3.2 
rust version: 1.75.0  
OS version: Macos 14.2.1
``` 
export component Example inherits Window {
    width: 100px;
    height: 100px;

    popup := PopupWindow {
        Rectangle { height:100%; width: 100%; background: yellow; }
        x: 20px; y: 20px; height: 50px; width: 50px;
    }

    TouchArea {
        height:100%; width: 100%;
        pointer-event(PointerEvent) => {
                    // Press the right mouse button
                    if (PointerEvent.button == PointerEventButton.right) {
                        popup.show();
                    }
                }
    }
}
```
question: When I press the the right mouse button, the window is appear，When I release it, this window disappears.
I want the function that when I click the right mouse button, the window will popup and don't disappear. How to achieve this function? Thanks!

**Comment by asuper0** [comment]
see [the doc](https://github.com/slint-ui/slint/blob/master/docs/reference/src/language/builtins/elements.md#properties-10)

> Properties
close-on-click (in bool): By default, a PopupWindow closes when the user clicks. Set this to false to prevent that behavior and close it manually using the close() function. (default value: true)

There has a [example of popup window](https://github.com/slint-ui/slint/tree/master/examples/todo)
